transmogrifier/sources/xml/datacite.py

```diff
@@ -339,6 +238,189 @@ def _get_contributors(cls, source_record: Tag) -> Iterator[timdex.Contributor]:
                     kind=contributor.get("contributorType") or "Not specified",
                 )
 
+    @classmethod
+    def get_dates(
+        cls,
+        source_record: Tag,
+    ) -> list[timdex.Date] | None:
+        dates = []
+        dates.extend(list(cls._get_publication_year(source_record)))
+        dates.extend(list(cls._get_dates(source_record)))
+        return dates or None
+
+    @classmethod
+    def _get_publication_year(cls, source_record: Tag) -> Iterator[timdex.Date]:
+        if publication_year := source_record.metadata.find(
+            "publicationYear", string=True
+        ):
+            publication_year = str(publication_year.string.strip())
+            if validate_date(
+                publication_year,
+                cls.get_source_record_id(source_record),
+            ):
+                yield timdex.Date(kind="Publication date", value=publication_year)
+        else:
+            logger.warning(
+                "Datacite record %s missing required Datacite field publicationYear",
+                cls.get_source_record_id(source_record),
+            )
```

Comment:
I see we log a warning, but this doesn't rise to the level of skipping the record?

Reply:
This is another `Datacite`-related warning like what @jonavellecuerdo commented about [here](https://github.com/MITLibraries/transmogrifier/pull/178/files/bdb2c4bd95c930683f7a5b6c2499fc88fe03eebb#r1612244336). I'll add it to the [issue](https://github.com/MITLibraries/transmogrifier/issues/180) because it should also likely be removed.